application/deed/views.py

```diff
@@ -34,8 +37,34 @@ def create():
         return error_message, status.HTTP_400_BAD_REQUEST
     else:
         deed.deed = deed_json
+
+        json_doc = {
+            "title_number": deed_json['title_number'],
+            "borrowers": []
+            }
+
         deed.token = Deed.generate_token()
+
+        date_check = functools.reduce(validation_checks,
+                                      deed_json['borrowers'], False)
+
+        if date_check:
+            abort(status.HTTP_400_BAD_REQUEST)
+
         try:
+            for borrower in deed_json['borrowers']:
+                borrower_doc = {
+                    "id": "",
+                    "forename": borrower['forename'],
+                    "middle_name": borrower['middle_name'],
+                    "surname": borrower['surname']
+                }
+
+                borrower_doc["id"] = borrowerService.extractBorrower(borrower)
```

Comment:
thats very confusing, should be something more along the lines of deserialiseBorrower